beamer/middleware.py

```diff
@@ -41,3 +51,221 @@ def middleware(method: RPCEndpoint, params: Any) -> RPCResponse:
         return response
 
     return middleware
+
+
+# The rate limiter middleware.
+#
+# In cases where an RPC returns HTTP 429 (Too many requests), we need to reduce
+# the rate at which we issue requests to that RPC. In order to do that, we
+# define the following operating modes:
+#
+# 1) normal operation
+# 2) rate limiting mode
+# 3) tapering mode
+#
+# In normal operation, the RPC accepts our every request and we issue requests
```

Comment:
`RPC accepts every request`

Reply:
> `RPC accepts every request`

I prefer to leave the `our` in there to make it more explicit that this is about RPC rate limiting us, not other possible clients. In other words, we may be experiencing rate limits, but other clients of the same RPC may issue requests just fine.

Reply:
the grammar sounds weird to me, do you mean `the RPC accepts every request of ours` or something like that?

Reply:
> the grammar sounds weird to me, do you mean `the RPC accepts every request of ours` or something like that?

Yes, that is the intended meaning. I've just tried several gramamr checkers and none reported an error.
Consider this:

- our every request
- your every request

I'm guessing you'd consider the second example (with "your") less strange, but from the grammar point of view, they are looking the same.